flight/targets/flyingf3/fw/pios_board.c

```diff
@@ -710,6 +710,16 @@ void PIOS_Board_Init(void) {
 	if (PIOS_BMP085_Test() != 0)
 		panic(5);
 #endif /* PIOS_INCLUDE_BMP085 && PIOS_INCLUDE_I2C */
+#if defined(PIOS_INCLUDE_ADC)
```

Comment:
Ok, I will change this asap. A few Questions:
1. Schould I change the PIOS_Assert also for the original rcflyer code?
2. Do I need to make a new pull request after I changed it?